src/cascadia/TerminalSettingsModel/EnumMappings.idl

```diff
@@ -23,5 +24,8 @@ namespace Microsoft.Terminal.Settings.Model
         static Windows.Foundation.Collections.IMap<String, Microsoft.Terminal.Core.CursorStyle> CursorStyle { get; };
         static Windows.Foundation.Collections.IMap<String, UInt16> FontWeight { get; };
         static Windows.Foundation.Collections.IMap<String, Microsoft.Terminal.Settings.Model.IntenseStyle> IntenseTextStyle { get; };
+
+        // Application State
+        static Windows.Foundation.Collections.IMap<String, Microsoft.Terminal.Settings.Model.InfoBarMessage> InfoBarMessage { get; };

```

Comment:
I don't think we need any of the EnumMappings changes. We use these to expose the `JSON_ENUM_MAPPER` from TerminalSettingsSerializationHelpers.h over to the settings UI.

Since we don't want to put these in the settings UI and `ApplicationState` is already accessible via the settings model, no need.